frontend/fingertips-frontend/components/organisms/Inequalities/inequalitiesHelpers.ts

```diff
@@ -185,6 +187,16 @@ export const getDynamicKeys = (
   return uniqueKeys;
 };
 
+const dashStyle = (index: number): DashStyleValue => {
+  if (index < 3) {
+    return 'Solid';
+  } else if (index < 8) {
+    return 'ShortDash';
+  } else {
+    return 'Dash';
+  }
```

Comment:
I think we could get rid of the else ifs to make neater
```suggestion
  if (index < 3) return 'Solid';
  if (index < 8) return 'ShortDash';
  return 'Dash';
```